reference/docs-conceptual/learn/shell/using-light-theme.md

```diff
@@ -0,0 +1,138 @@
+default colors don't set the background, using light foreground colors on a light background
+produces unreadable text.
+
+**PSReadLine** allows you to define colors for 18 different syntax elements. You can view the
+current settings using the `Get-PSReadLineOption` cmdlet.
+
+```Output
+EditMode                               : Windows
+AddToHistoryHandler                    : System.Func`2[System.String,System.Object]
+HistoryNoDuplicates                    : True
+HistorySavePath                        : C:\Users\user1\AppData\Roaming\Microsoft\Wind...
+HistorySaveStyle                       : SaveIncrementally
+HistorySearchCaseSensitive             : False
+HistorySearchCursorMovesToEnd          : False
+MaximumHistoryCount                    : 4096
+ContinuationPrompt                     : >>
+ExtraPromptLineCount                   : 0
+PromptText                             : {> }
+BellStyle                              : Audible
+DingDuration                           : 50
+DingTone                               : 1221
+CommandsToValidateScriptBlockArguments : {ForEach-Object, %, Invoke-Command, icm...}
+CommandValidationHandler               :
+CompletionQueryItems                   : 100
+MaximumKillRingCount                   : 10
+ShowToolTips                           : True
+ViModeIndicator                        : None
+WordDelimiters                         : ;:,.[]{}()/\|^&*-=+'"-—―
+AnsiEscapeTimeout                      : 100
+PredictionSource                       : HistoryAndPlugin
+PredictionViewStyle                    : InlineView
+CommandColor                           : "`e[93m"
+CommentColor                           : "`e[32m"
+ContinuationPromptColor                : "`e[37m"
+DefaultTokenColor                      : "`e[37m"
+EmphasisColor                          : "`e[96m"
+ErrorColor                             : "`e[91m"
+InlinePredictionColor                  : "`e[38;5;238m"
+KeywordColor                           : "`e[92m"
+ListPredictionColor                    : "`e[33m"
+ListPredictionSelectedColor            : "`e[48;5;238m"
+MemberColor                            : "`e[97m"
+NumberColor                            : "`e[97m"
+OperatorColor                          : "`e[90m"
+ParameterColor                         : "`e[90m"
+SelectionColor                         : "`e[30;47m"
+StringColor                            : "`e[36m"
+TypeColor                              : "`e[37m"
+VariableColor                          : "`e[92m"
+```
+
+The color settings are stored as strings containing ANSI escape sequences that change the color in
+your terminal. Using the `Set-PSReadLineOption` cmdlet you can change the colors to values that work
+better for a light-colored background.
+
+## Defining colors for a light theme
+
+The original PowerShell ISE can be configured to use a light theme for both the editor and console
```

Comment:
I'm not sure "original" is needed here. It might be beneficial to mention ISE mode in VS Code and link to the [theme definition file](https://github.com/PowerShell/vscode-powershell/blob/main/themes/theme-psise/theme.json), which contains colors and tokens in JSON.

```suggestion
The PowerShell ISE can be configured to use a light theme for both the editor and console
```